bin/configs/spring-http-interface-useHttpServiceProxyFactoryInterfacesConfiguration.yaml

```diff
@@ -0,0 +1,21 @@
+generatorName: spring
+library: spring-http-interface
+outputDir: samples/client/petstore/spring-http-interface-useHttpServiceProxyFactoryInterfacesConfigurator
+inputSpec: modules/openapi-generator/src/test/resources/3_0/spring/petstore-with-fake-endpoints-models-for-testing.yaml
+templateDir: modules/openapi-generator/src/main/resources/JavaSpring
+additionalProperties:
+  artifactId: spring-http-interface-useHttpServiceProxyFactoryInterfacesConfigurator
+  snapshotVersion: "true"
+  hideGenerationTimestamp: "true"
+  # documentation provider should be ignored
+  documentationProvider: "springdoc"
+  # annotation provider should be ignored
+  annotationLibrary: "swagger2"
+  # validation should be ignored
+  useBeanValidation: "true"
+  performBeanValidation: "true"
+  useHttpServiceProxyFactoryInterfacesConfigurator: "true"
+  # needed until updating the default parentVersion (3.1.3) to 3.2+
```

Comment:
Is this still true after removing the two imports mentioned in my other comment?

Reply:
Good point, removed the parent overrides!